docs/cli/dnx-migration.md

```diff
@@ -0,0 +1,155 @@
+# Migrating from DNX to .NET Core CLI
+
+## Overview
+With RC1 release of .NET Core and ASP.NET Core 1.0, we introduced DNX tooling to the world. With RC2 release of .NET 
+Core and ASP.NET Core 1.0 we are transitioning to the .NET Core CLI.
+
+As a slight refresher, let's recap what DNX was about. DNX was a runtime and a toolset used to build .NET Core and, 
+more specifically, ASP.NET Core 1.0 applications. It consisted of 3 main pieces:
+
+1. DNVM - an install script to obtaining new DNX-es
+2. DNX (Dotnet Execution Runtime) - the runtime that executes your code
+3. DNU (Dotnet Developer Utility) - tooling for managing dependencies, building and publishing your apps
+
+With the introduction of the CLI, all of the above are part of a single toolset now, and thus DNX has been 
+effectively discontinued. However, since it was available for a long time, you might have projects that you 
+built using it that you would want to move off to the new CLI tooling. 
+
+This migration guide will cover the essentials on how to migrate projects off of DNX and onto .NET Core CLI. If you are just 
+starting a project on .NET Core from scratch, you can freely skip this document. 
+
+## Main changes in the tooling
+There are some general changes in the tooling that should be outlined first. 
+
+### No more DNVM
+DNVM, short for *DotNet Version Manager* was a bash/PowerShell script used to install a DNX on your machine. It helped
+users get the DNX they need from the feed they specified (or default ones) as well as mark a certain DNX "active", which 
+would put it on the $PATH for the given session. This would allow you to use the various tools.
+
+DNVM was discontinued because its feature set is made redundant by changes coming in the CLI.
+
+CLI comes packaged in two main ways, as was specified in the [intro document](overview.md#installation):
+
+1. Native installers for a given platform
+2. Install script for other situations (like CI servers)
+
+Given this, the DNVM install features are not needed. But what about the runtime selection features? 
+
+You reference a runtime in your `project.json` by adding a package of a certain version to your dependencies. With this change
```

Comment:
Along these lines, we may want to point out that the sdk/version element in global.json (which helped VS tooling know which version of DNU to use) should not be used anymore. If uses had this present previously and leave it in, it will confuse CLI VS tooling. It's a bit more IDE-tooling specific than most of this article, but I expect a lot of folks will run into it while migrating.